cpp/CMakeLists.txt

```diff
@@ -369,11 +390,7 @@ include_directories("${CMAKE_BINARY_DIR}/include"
                     "${CMAKE_BINARY_DIR}/include/jit"
                     "${CMAKE_SOURCE_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/src"
-                    "${ARROW_INCLUDE_DIR}"
-                    "${ZLIB_INCLUDE_DIRS}"
-                    "${Boost_INCLUDE_DIRS}"
-                    "${RMM_INCLUDE}"
-                    "${DLPACK_INCLUDE}")
+                    "${ARROW_INCLUDE_DIR}")
```

Comment:
If we're using the arrow target we shouldn't need to include this

Reply:
The arrow install is so specific, I was going to save moving that to targets to the follow up PR for Friday as it has such a custom set of compile details. I have left arrow alone on this pass and not converted it to CPM. Is that ok? Should I solve that now? I can do that today, but I'd like to get this in with code freeze.

Reply:
We should handle Arrow as a dependency using CPM similar to other dependencies and if not all the dependencies are being handled via CPM then we shouldn't put it in before the release.